File: class_tool/src/main/java/com/tempsfteam/class_tool/dto/ClasseDTO.java
package com.tempsfteam.class_tool.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author hypocodeemia
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class ClasseDTO {
    private Integer classeId;

    private String name;

    private Integer schoolId;
}

File: class_tool/src/main/resources/static/src/models/dto/classeDTO.js
export default {
    name: 'classeDTO',
    title: 'ClasseDTO',
    fields: [
        { name: 'classeId', label: '班级ID', type: 'number' },
        { name: 'name', label: '班级名称', type: 'string', required: true },
        { name: 'schoolId', label: '学校ID', type: 'number', required: true },
        { name: 'grade', label: '年级', type: 'string' }
    ]
};